libbeat/cmd/instance/beat.go

```diff
@@ -147,6 +147,7 @@ func initRand() {
 // instance.
 // XXX Move this as a *Beat method?
 func Run(settings Settings, bt beat.Creator) error {
+	setUmaskWithSettings(settings)
```

Comment:
Should we lock failures? E.g. add an ErrNotImplemented and only log `if err != nil && err != ErrNotImplemented`.

Reply:
Yes, I think this is a good idea. Thanks, will implement.